backend/config/config.go

```diff
@@ -228,6 +228,41 @@ func NewConfig(appConfigFilename string, accountsConfigFilename string) (*Config
 	return config, nil
 }
 
+// SetBtcOnly sets non-bitcoin accounts in the config to false
+func (config *Config) SetBtcOnly() {
+	config.appConfig.Backend.LitecoinP2WPKHP2SHActive = false
+	config.appConfig.Backend.LitecoinP2WPKHActive = false
+	config.appConfig.Backend.EthereumActive = false
+}
+
+// SetBTCElectrumServers sets the BTC configuration to the provided electrumIP and electrumCert
+func (config *Config) SetBTCElectrumServers(electrumAddress, electrumCert string) {
+	var btcBaseCoinConfig = btcCoinConfig{
```

Comment:
var name invalid ;) you can also skip a variable here and directly assign the struct below.